rust/src/lib.rs

```diff
@@ -48,12 +49,8 @@ impl GraphWrapper {
         }
     }
 
-    pub fn get_modules(&self) -> HashSet<String> {
-        self._graph
-            .all_modules()
-            .visible()
-            .names_as_strings()
-            .collect()
+    pub fn get_modules(&self) -> GraphModuleNames {
+        self._graph.all_modules().visible().names().collect()
```

Comment:
There is still a copy going on here, but now we only have to copy the symbols and not the entire strings. It's quite a lot faster.